File: src/main/java/Related.java
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

import java.nio.ByteBuffer;
import java.nio.channels.SelectionKey;

@Getter @Setter
@AllArgsConstructor
public class Related {
    private SelectionKey relatedChanel = null;
    private boolean isClient;
    private int countMes;
    private ByteBuffer writeThis;
    private ByteBuffer readThis;
    private int connectionNum;
}

File: src/main/kotlin/Related.kt
import java.nio.ByteBuffer
import java.nio.channels.SelectionKey

data class Related(
        var relatedChanel : SelectionKey? = null,
        var isClient : Boolean,
        var countMes : Int,
        var writeThis : ByteBuffer,
        var readThis : ByteBuffer,
        val connectionNum : Int
)
